src/shared/components/hmcts-main-wrapper/hmcts-main-wrapper.component.html

```diff
@@ -0,0 +1,27 @@
+<a *ngIf="backLink && !backAction" [routerLink]="backLink" class="govuk-back-link">Back</a>
```

Comment:
the same with hmcts-main-wrapper, it is available in common-lib

Reply:
Again @elviani I have changed it slightly. I will do that later if you alright